bindings/nodejs/tests/testcontainers.test.mjs

```diff
@@ -0,0 +1,130 @@
+import { GenericContainer, getContainerRuntimeClient } from 'testcontainers'
```

Comment:
It's better to have those code seperated in another PR.

And opendal has an integration tests setup for all bindings. It might not a good idea to introduce `testcontainers` in nodejs binding.

Reply:
Got it, I'll make sure to keep this PR concise.